src/services/services.ts

```diff
@@ -508,7 +508,9 @@ function addSyntheticNodes(nodes: Node[], pos: number, end: number, parent: Node
                 if (hasTabstop(parent)) {
                     continue;
                 }
-                Debug.fail(`Did not expect ${Debug.formatSyntaxKind(parent.kind)} to have an Identifier in its trivia`);
+                // TODO: Not sure why this is here. When I try to make text change that contains only a property assigment this kicks in

```

Comment:
This is a guard against constructing invalid trees; usually this indicates a bug the parser or in something that creates nodes and sets their position.

I've previously advocated for removing this check, but in every case it's fired, it has turned out to be a real bug somewhere else that had to be fixed.

Reply:
Rewmoved. The real reason was that I was using a node in two positions and the formatter did not like that.